src/Features/header/components/Cities/Cities.tsx

```diff
@@ -1,40 +1,66 @@
 import { FiNavigation } from 'react-icons/fi';
-import { ReactElement, useMemo, useCallback } from 'react';
+import { ReactElement, useMemo, useState, useEffect, useCallback } from 'react';
 
-import { useFetchCityListQuery } from '../Sidebar/SidebarSlice';
-import { LoadingSpinner } from '../../../../components/LoadingSpinner/LoadingSpinner';
+import { LoadingSpinner } from '../../../../components/LoadingSpinner';
+import { useLazyFetchCityListQuery } from '../Sidebar/SidebarSlice';
 import { CityList } from './Cities.types';
 
 export const Cities = (): ReactElement => {
-  const { data, isLoading } = useFetchCityListQuery('');
+  const [page, setPage] = useState(0);
+  const [offset] = useState(8);
```

Comment:
Here it will be more logical to use just constant from. Create it and use it here. State is something that changes, here we don't see it